currency_converter.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/python3
+
+import argparse
+import json
+import sys
+
+from forex_python.converter import CurrencyCodes
+from forex_python.converter import CurrencyRates
+from forex_python.converter import RatesNotAvailableError
+
+
+code_dictionary = \
+    { u'$':   'USD'
+    , u'Fr.': 'CHF'
+    , u'Ft':  'HUF'
+    , u'HK$': 'HKD'
+    , u'Kr':  'DKK'
+    , u'Kč':  'CZK'
+    , u'L':   'RON'
+    , u'NZ$': 'NZD'
+    , u'R$':  'BRL'
+    , u'R':   'RUB'
+    , u'RM':  'MYR'
+    , u'Rp':  'IDR'
+    , u'S$':  'SGD'
+    , u'W':   'KRW'
+    , u'kn':  'HRK'
+    , u'kr':  'NOK'
+    , u'zł':  'PLN'
+    , u'£':   'GBP'
+    , u'¥':   'JPY'
+    , u'฿':   'THB'
+    , u'₪':   'ILS'
+    , u'€':   'EUR'
+    , u'₱':   'PHP'
+    , u'₹':   'INR'
+    }
+
+def main(argv):
+
+    parser = argparse.ArgumentParser(description='Currency converter.')
+    # Options:
+    # --amount
+    # --input_currency
+    # --output_currency
+    # NOTE: --help option is implicit with 'argparse'.
+    parser.add_argument("--amount", metavar='AMOUNT', type=float,
+        dest='amount', required=True, help='amount to convert')
```

Comment:
Oh, I care about spaces. It's totally unnecessary to indent this much. It adds just too much spaces, plus the longer the function name is, it's more ineficient.
Btw. [PEP 8 - Indentation](https://www.python.org/dev/peps/pep-0008/#indentation) supports this style of formatting too.